tests/NumericHelperTest.php

```diff
@@ -66,4 +66,28 @@ public function testNormalizeNumberWithIncorrectType(): void
         $this->expectException(\InvalidArgumentException::class);
         NumericHelper::normalize([]);
     }
+
+    public function dataIsInteger(): array
+    {
+        return [
+            ['42', true],
+            ['-42', true],
+            ['0', true],
```

Comment:
What about case with value `'042'`? It will be convert to `'42'` and `isInteger` returns false. Is it correct?

Reply:
Fixed